react/MiniCart.js

```diff
@@ -61,6 +61,13 @@ export class MiniCart extends Component {
       to: detailUrl
     })
   }
+  
+  get itemsQuantity() {
+    const { orderFormContext: { orderForm }} = this.props
+    if (!orderForm || !orderForm.items) return 0
+    const isParentItem = ({ parentItemIndex, parentAssemblyBinding }) => parentItemIndex == null && parentAssemblyBinding == null
```

Comment:
⛏ valeria a pena quebrar a linha aqui

```suggestion
    const isParentItem = ({ parentItemIndex, parentAssemblyBinding }) =>
       parentItemIndex == null && parentAssemblyBinding == null
```